pkg/investigations/chgm/chgm.go

```diff
@@ -110,6 +98,37 @@ func Investigate(r *investigation.Resources) error {
 	return r.PdClient.EscalateAlertWithNote(notes.String())
 }
 
+// handleBlockedEgress sends limited support or service logs appropriate to the blocked egresses
+func handleBlockedEgress(cluster *cmv1.Cluster, ocmCli ocm.Client, egressFailures string) error {
+	// These two URLs are absolutely critical for customers to keep open.
+	// Without them, we can't guarantee SLA, therefore these two should result in limited support.
+	urlsToLimitedSupport := []string{"pagerduty.com", "nosnch.in"}
+
+	setLS := false
+	for _, urlsToLimitedSupport := range urlsToLimitedSupport {
+		if strings.Contains(egressFailures, urlsToLimitedSupport) {
+			setLS = true
+			break
+		}
+	}
+
+	if setLS {
+		err := ocmCli.PostLimitedSupportReason(createEgressLS(egressFailures), cluster.ID())
+		if err != nil {
+			return err
+		}
+		metrics.Inc(metrics.LimitedSupportSet, investigationName, "EgressBlocked")
```

Comment:
thought: we often run into issues with `metrics.Inc` due to the nature of it accepting everything, we should wrap the metric increases into functions to properly check parameters.

Reply:
agreed